### PR TITLE
Properly set OS of nodes when want_sles12sp2 is set

### PR DESCRIPTION
**Pull request by jsuchome**
We need this to pass current SP2 gating before the final switch to SP2 (than want_sles12sp2 will be removed)

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -792,6 +792,7 @@ function get_sles12plus_node()
 {
     local target="suse-12.0"
     iscloudver 6plus && target="suse-12.1"
+    iscloudver 7plus && [[ $want_sles12sp2 ]] && target="suse-12.2"
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: but if not `$want_sles12sp2` then target will be `suse-12.0` for 7plus. Change 793 default and remove 794?

**Comment by abel-navarro** [comment]
+1

Note: SP2 is going to be setup for admin and non-admin nodes. Admin node will be able to choose SP1 with https://github.com/SUSE-Cloud/automation/pull/1033

**Comment by sayalilunkad** [comment]
LGTM, +1

**Comment by dirkmueller** [comment]
+1

**Pull request merged**